web/sass-files/sass/partials/_post.scss

```diff
@@ -106,6 +106,39 @@ body.ios {
 	}
 }
 
+.file-overlay {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	background-color: rgba(0, 0, 0, 0.6);
+	text-align: center;
+	color: #FFF;
+	display: table;
+	font-size: 1.7em;
+	font-weight: 600;
+	z-index: 6;
+
+	> div {
+		display: table-cell;
+		vertical-align: middle;
+	}
+
+	.fa {
+		display: block;
+		font-size: 2em;
+		margin: 0 0 0.3em;
+	}
+
+	.invisible {
+		visibility: hidden;
+	}
+	.visible {
+		visibility: visible;
```

Comment:
Thanks for the comments @ralder I think you're right for both of these let me take a look and see about changing these.